<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>equipementmed</class>
 <widget class="QDialog" name="equipementmed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MED++</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1291</width>
     <height>721</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane { /* The tab widget frame */
    border-top: 2px solid #C2C7CB;
}

QTabWidget::tab-bar {
    left: 5px; /* move to the right by 5px */
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 2px solid #C4C4C3;
    border-bottom-color: #C2C7CB; /* same as the pane color */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 8ex;
    padding: 2px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
}

QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-bottom-color: #C2C7CB; /* same as pane color */
}

QTabBar::tab:!selected {
    margin-top: 2px; /* make non-selected tabs look smaller */
}</string>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter Equipements</string>
    </attribute>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1281</width>
       <height>701</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget{background-image: url(:/AjouterEqui.png);}
</string>
     </property>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>390</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <widget class="QPushButton" name="save">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-10</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>1150</x>
        <y>60</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <widget class="QPushButton" name="retour">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="a">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>270</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="b">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>330</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="c">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>390</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="d">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>440</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Modifier Equipements</string>
    </attribute>
    <widget class="QWidget" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1281</width>
       <height>711</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget_6
{
	background-image: url(:/ModifierEqui.png);
}</string>
     </property>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>170</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}


</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="retour_2">
      <property name="geometry">
       <rect>
        <x>1150</x>
        <y>60</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>170</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>1124</x>
        <y>392</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}


</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>180</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width:2px;
border-style:solid;
border-color:darkblue;

border-radius: 3px;
padding: 1px 18px 1px 3px;
min-width: 6em;

background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                       stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                       stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);


padding-top: 3px;
padding-left: 4px;

subcontrol-origin: padding;
subcontrol-position: top right;
width: 15px;

border-left-width: 1px;
border-left-color: darkgray;
border-left-style: solid; 
border-top-right-radius: 3px; 
border-bottom-right-radius: 3px;

top: 1px;
left: 1px;

border: 2px solid darkgray;
selection-background-color: lightgray;

</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>180</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}


</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="a2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>280</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="d2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>450</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="c2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>390</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="b2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>330</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Supprimer Equipements</string>
    </attribute>
    <widget class="QWidget" name="widget_7" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1281</width>
       <height>691</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget_7
{
	background-image: url(:/SuppEqui.png);
}</string>
     </property>
     <widget class="QWidget" name="widget_8" native="true">
      <property name="geometry">
       <rect>
        <x>1150</x>
        <y>60</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_9" native="true">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>650</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <widget class="QPushButton" name="Delete">
       <property name="geometry">
        <rect>
         <x>-20</x>
         <y>0</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>380</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width:2px;
border-style:solid;
border-color:darkblue;

border-radius: 3px;
padding: 1px 18px 1px 3px;
min-width: 6em;

background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                       stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                       stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);


padding-top: 3px;
padding-left: 4px;

subcontrol-origin: padding;
subcontrol-position: top right;
width: 15px;

border-left-width: 1px;
border-left-color: darkgray;
border-left-style: solid; 
border-top-right-radius: 3px; 
border-bottom-right-radius: 3px;

top: 1px;
left: 1px;

border: 2px solid darkgray;
selection-background-color: lightgray;

</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="mm">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>420</y>
        <width>391</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>450</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#label_2 
{

	color: rgb(79, 154, 211);
}</string>
      </property>
      <property name="text">
       <string>Raisons de suppression : </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pa">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>590</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:rgb(79, 154, 211);
border-radius:7px;
}
</string>
      </property>
      <property name="text">
       <string>IMPRIMER</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Afficher Equipements</string>
    </attribute>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1281</width>
       <height>711</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget_4
{
	background-image: url(:/AfficherEqui.png);
}</string>
     </property>
     <widget class="QTableView" name="taba">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>1281</width>
        <height>461</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView QTableCornerButton::section {
    background: red;
    border: 2px outset red;
}
QTableView {
    selection-background-color: qlineargradient(x1: 0, y1: 0, x2: 0.5, y2: 0.5,
                                stop: 0 #FF92BB, stop: 1 gray);
}
 QScrollBar:vertical {
     border: 2px solid grey;
     background: #32CC99;
     width: 15px;
     margin: 22px 0 22px 0;
 }
 QScrollBar::handle:vertical {
     background: white;
     min-height: 20px;
 }
 QScrollBar::add-line:vertical {
     border: 2px solid grey;
     background: #32CC99;
     height: 20px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }

 QScrollBar::sub-line:vertical {
     border: 2px solid grey;
     background: #32CC99;
     height: 20px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }
 QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
     border: 2px solid grey;
     width: 3px;
     height: 3px;
     background: white;
 }

 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
     background: none;
 }

</string>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
     </widget>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>1150</x>
        <y>60</y>
        <width>120</width>
        <height>41</height>
       </rect>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-10</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Trier Par : </string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
    width: 13px;
    height: 13px;
}</string>
      </property>
      <property name="text">
       <string>A-Z</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
    width: 13px;
    height: 13px;
}</string>
      </property>
      <property name="text">
       <string>Z-A</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>200</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
    width: 13px;
    height: 13px;
}</string>
      </property>
      <property name="text">
       <string>quantitée ↑</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>200</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
    width: 13px;
    height: 13px;
}</string>
      </property>
      <property name="text">
       <string>quantitée ↓</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>180</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>180</y>
        <width>84</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}
QPushButton:open { /* when the button has its menu open */
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton::menu-indicator {
    image: url(menu_indicator.png);
    subcontrol-origin: padding;
    subcontrol-position: bottom right;
}

QPushButton::menu-indicator:pressed, QPushButton::menu-indicator:open {
    position: relative;
    top: 2px; left: 2px; /* shift the arrow by 2 px */
}</string>
      </property>
      <property name="text">
       <string>Chercher</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
